app/controllers/participate/login.js

```diff
@@ -24,7 +24,7 @@ export default Ember.Controller.extend({
           this.transitionToRoute('participate.survey.consent');
         })
         .catch((e) => {
-          if (e.status === 404) {
+          if (e.status === 404 || e.status === 401) {
```

Comment:
We may be looking at the same beforeModel code; possibly relevant:
https://github.com/CenterForOpenScience/isp/pull/102/files